Xamarin.Forms.Core.UnitTests/CheckBoxUnitTests.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using NUnit.Framework;
+
+namespace Xamarin.Forms.Core.UnitTests
+{
+	[TestFixture]
+	public class CheckBoxUnitTests : BaseTestFixture
+	{
+		[Test]
+		public void TestConstructor ()
```

Comment:
Formatting - drop the space between the method name and the opening parenthesis.